systemverilog-plugin/UhdmAst.cc

```diff
@@ -2012,7 +2012,7 @@ void UhdmAst::process_design()
                 current_node->children.push_back(pair.second);
             }
         } else {
-            log_warning("Removing unused module: %s from the design.\n", pair.second->str.c_str());
+            log_warning("Removing unparameterized module: %s from the design.\n", pair.second->str.c_str());
```

Comment:
What does it actually mean that the module is unparameterized?

Reply:
This is ``unelaborated`` model from Surelog, so one that doesn't have parameters replaced, genscopes unrolled etc.

I actually changed message to ``unelaborated``.

Reply:
Ok, now it's clear, thanks!